Sources/Commands/SwiftTool.swift

```diff
@@ -677,6 +687,9 @@ public class SwiftTool<Options: ToolOptions> {
         if verbosity != .concise {
             args.append("-v")
         }
+        if numThreads != 0 {
+            args.append("-j(\(numThreads))")
```

Comment:
What happens if you pass -j0 to llbuild? Maybe it's better to make numThreads optional if 0 doesn't have semantic meaning across tools.